python/ctsm/utils.py

```diff
@@ -207,9 +207,8 @@ def _handle_config_value(var, default, item, is_list, convert_to_type,
     if allowed_values is not None:
         for v in var:
             if v not in allowed_values:
-                print('v in var not in allowed_values', v)
-                errmsg = "{} is not an allowed value for {} in .cfg file\n" \
-                         "Allowed values: {}".format(v, item, allowed_values)
+                print('v = ', v, ' in var not in allowed_values')
+                errmsg = '{} is not an allowed value for {} in .cfg file. Check allowed_values'.format(v, item)
```

Comment:
Changed this because I could not get the unit test to work when `allowed_values` was part of `errmsg`. I hope you consider this fine, too, and I'm also open if not.